base_layer/core/src/chain_storage/db_transaction.rs

```diff
@@ -78,6 +78,13 @@ impl DbTransaction {
         self.insert(DbKeyValuePair::UnspentOutput(hash, Box::new(utxo)));
     }
 
+    #[cfg(test)]
```

Comment:
This might be a useful function in non-test cases too. For example, if you've already calculated the hash somewhere, and want to save another hashes calc.